### ISSUE TITLE
Option to display values for GPS2 instrument panel (the selectable one)

### ISSUE THREAD

**Issue by pompecukor** [mavlink/qgroundcontrol#9435]
On the user configurable display values, would be nice to be able to also show GPS2 (count and HDOP).
Right now only possible to show for GPS1. one, but that is already show on the horizontal bar at the top,

For us this is mainly useful for when we are testing the aircraft. On QGC there is just no was to make GPS2 information to show.
BTW same goes for secondary Airspeed sensor. 

Thank you.

**Comment by Davidsastresas** [comment]
Partially fixed in #9437. For the airspeed matter I think we need development from mavlink and autopilot firmware as well, as I couldn't see where is currently sent the second airspeed sensor info.

**Comment by Davidsastresas** [comment]
Okey, apparently Airspeed 2 is sent right now from Ardupilot by a "named float" mavlink message. This is a workaround until we have support for it from mavlink, so we should really open an issue in mavlink, and when we have proper support in mavlink for this add it to QGC.

**Comment by eduardkieser** [comment]
Hi @Davidsastresas, I have a related question. I have implemented a sensor driver in lua for an unsupported temperature sensor. I would like to view the values of that sensor in the "Instrument Panel". The message is being sent as a `named_float`, and on mission planner I can configure the quick view to display the value. Is there any way that I can view named floats in the QGC gui?

**Comment by Davidsastresas** [comment]
I don't think we have support for that right now, but it can be added. Can you open a separate issue for that matter? thanks!

**Comment by eduardkieser** [comment]
sure thing. [10564](https://github.com/mavlink/qgroundcontrol/issues/10564).
